docs/tutorials/java/mxnet_java_on_intellij.md

```diff
@@ -167,5 +166,5 @@ java -cp target/javaMXNet-1.0-SNAPSHOT.jar:target/dependency/* mxnet.App
 For more information about MXNet Java resources, see the following:
 
 * [Java Inference API](https://mxnet.incubator.apache.org/api/java/infer.html)
-* [Java Inference Examples](https://github.com/apache/incubator-mxnet/tree/java-api/scala-package/examples/src/main/java/org/apache/mxnetexamples/infer/)
+* [Java Inference Examples](https://github.com/apache/incubator-mxnet/tree/java-api/scala-package/examples/src/main/java/org/apache/mxnetexamples/javaapi/infer)
```

Comment:
Please change the URL to `master`

https://github.com/apache/incubator-mxnet/tree/master/scala-package/examples/src/main/java/org/apache/mxnetexamples/javaapi/infer